apps/asctb-reporter/src/app/components/omap-controls/omap-controls.component.ts

```diff
@@ -1,22 +1,42 @@
-import { Component, EventEmitter, Input, Output } from '@angular/core';
+import { Component, input, output } from '@angular/core';
+import { MatButtonModule } from '@angular/material/button';
+import { MatCheckboxModule } from '@angular/material/checkbox';
+import { MatExpansionModule } from '@angular/material/expansion';
+import { MatIconModule } from '@angular/material/icon';
+import { MatSelectModule } from '@angular/material/select';
+import { MatTooltipModule } from '@angular/material/tooltip';
+import { HraCommonModule } from '@hra-ui/common';
 import { OmapConfig } from '../../models/omap.model';
 import { Error } from '../../models/response.model';
 
 @Component({
   selector: 'app-omap-controls',
-  standalone: false,
+  imports: [
+    MatExpansionModule,
+    MatSelectModule,
+    MatIconModule,
+    MatTooltipModule,
+    MatButtonModule,
+    MatCheckboxModule,
+    HraCommonModule,
+  ],
   templateUrl: './omap-controls.component.html',
   styleUrl: './omap-controls.component.scss',
 })
 export class OmapControlsComponent {
-  @Input() omaps: OmapConfig = { organsOnly: false, proteinsOnly: false };
-  @Input() error!: Error;
+  /** OMAP configuration */
+  protected readonly omaps = input<OmapConfig>({ organsOnly: false, proteinsOnly: false });
 
-  @Output() readonly updateConfig = new EventEmitter<OmapConfig>();
+  /** Error information */
+  protected readonly error = input<Error>();
 
+  /** Output for OMAP configuration update */
+  protected readonly updateConfig = output<OmapConfig>();
```

Comment:
Can be removed and replaced with `[(omaps)]="omapsSignal"` or `(omapsChange)="doSomething($event)"`